doc/dev/dashboard/index.rst

```diff
@@ -0,0 +1,10 @@
+======================================
+Ceph Dashboard Developer Documentation
+======================================
+
+.. rubric:: Contents
+
+.. toctree::
+   :maxdepth: 1 
+
+   UI Design Principles <ui_goals>
```

Comment:
Note that we already have a document titled "Ceph Dashboard Developer Documentation" located at `/doc/dev/developer_guide/dash-devel.rst` - how about adding a section "Design Documents" to that document and linking to the Design Principles from there?

Reply:
Or how about simply adding the content of this PR as a intro chapter to that document?

Reply:
OK - will store the content under /doc/dev/dashboard/ and link to it from dash-devel. Not keen on adding more content to dash-devel - it's already >2000 lines